static/app/views/insights/browser/webVitals/components/webVitalMetersWithIssues.tsx

```diff
@@ -0,0 +1,337 @@
+import {IconIssues} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import useOrganization from 'sentry/utils/useOrganization';
+import {PerformanceBadge} from 'sentry/views/insights/browser/webVitals/components/performanceBadge';
+import {VITAL_DESCRIPTIONS} from 'sentry/views/insights/browser/webVitals/components/webVitalDescription';
+import {WEB_VITALS_METERS_CONFIG} from 'sentry/views/insights/browser/webVitals/components/webVitalMeters';
+import {
+  getIssueQueryFilter,
+  useWebVitalsIssuesQuery,
+} from 'sentry/views/insights/browser/webVitals/queries/useWebVitalsIssuesQuery';
+import {MODULE_DOC_LINK} from 'sentry/views/insights/browser/webVitals/settings';
+import {
+  type ProjectScore,
+  WEB_VITAL_PERFORMANCE_ISSUES,
+  type WebVitals,
+} from 'sentry/views/insights/browser/webVitals/types';
+
+export type ProjectData = {
+  'p75(measurements.cls)': number;
+  'p75(measurements.fcp)': number;
+  'p75(measurements.inp)': number;
+  'p75(measurements.lcp)': number;
+  'p75(measurements.ttfb)': number;
+};
+
+type Props = {
+  onClick?: (webVital: WebVitals) => void;
+  projectData?: ProjectData[];
+  projectScore?: ProjectScore;
+  showTooltip?: boolean;
+  transaction?: string;
+};
+
+export default function WebVitalMetersWithIssues({
+  onClick,
+  projectData,
+  projectScore,
+  showTooltip = true,
+}: Props) {
+  const theme = useTheme();
+  if (!projectScore) {
+    return null;
+  }
+
+  const webVitalsConfig = WEB_VITALS_METERS_CONFIG;
+
+  const webVitals = Object.keys(webVitalsConfig) as WebVitals[];
+  const colors = theme.chart.getColorPalette(3);
+
+  const renderVitals = () => {
+    return webVitals.map((webVital, index) => {
+      const webVitalKey: keyof ProjectData = `p75(measurements.${webVital})`;
+      const score = projectScore[`${webVital}Score`];
+      const meterValue = projectData?.[0]?.[webVitalKey];
+
+      if (!score) {
+        return null;
+      }
+
+      return (
+        <VitalMeter
+          key={webVital}
+          webVital={webVital}
+          showTooltip={showTooltip}
+          score={score}
+          meterValue={meterValue}
+          color={colors[index]!}
+          onClick={onClick}
+        />
+      );
+    });
+  };
+
+  return (
+    <Container>
+      <Flex>{renderVitals()}</Flex>
+    </Container>
+  );
+}
+
+type VitalMeterProps = {
+  color: string;
+  meterValue: number | undefined;
+  score: number | undefined;
+  showTooltip: boolean;
+  webVital: WebVitals;
+  isAggregateMode?: boolean;
+  onClick?: (webVital: WebVitals) => void;
+};
+
+function VitalMeter({
+  webVital,
+  score,
+  meterValue,
+  onClick,
+  isAggregateMode = true,
+  showTooltip = true,
+}: VitalMeterProps) {
+  const organization = useOrganization();
+  const [isIssuesButtonHovered, setIsIssuesButtonHovered] = useState(false);
```

Comment:
Whoa, why manually keep track of hover state?

Reply:
This is a little gross but I couldn't find a more elegant way to handle this.

Basically, when hovering over the issue button, by default the `InteractionStateLayer` on the meter component will activate. This looks a little weird because the meter and the issue button within are highlighted simultaneously:
<img width="251" alt="image" src="https://github.com/user-attachments/assets/e9ee17f8-f0a3-4307-903a-52955d87ac05" />

I added the manual tracking to work around this so I can exclusively highlight the issue button and not the entire meter. 

Alternatively, maybe we just stop using `InteractionStateLayer` (I heard it might be going away) and use something else to handle highlighting.

Let me know if you have any other suggestions!

Reply:
Dang! What about doing an `event.stopPropagation()` on the hover on the button? If that works, it's still better. Also worth adding a code comment to explain why this is happening, since it's surprising!

Reply:
Ok!